_docs/deployment/logs.md

```diff
@@ -53,16 +53,16 @@ Logs are currently retained for 180 days, and you will only see data for applica
 
 After logging in, you'll see the App Overview dashboard.
 
-{% asset app-overview.png alt="App Overview dashboard" %}
+!["App Overview dashboard"]({{site.baseurl}}/assets/images/content/app-overview.png)
```

Comment:
Idea: alternative syntax that I gravitated towards as being more readable, but totally my own personal preference:
`!["App Overview dashboard"]({{ "/assets/images/content/app-overview.png" | relative_url }})`